OpsCenter_6.8_Release_Notes.md

```diff
@@ -3,6 +3,8 @@
 # Release Notes for OpsCenter 6.8.35
 22 Mar 2024
 
+:warning: Customers who are upgrading the OpsCenter to 6.8.35 from older versions may face issue with agent installation. This issue can be addressed by altering the cluster table in lcm.db. For details please refer to OPSC-17503.
```

Comment:
Summarizing the discussion from stand up:
If there is a release tomorrow to get the fix out, change this warning to point to the newer version to avoid the agent install problem.
If there is not a release, we need to have the work around steps somewhere. Either here, or possibly a support knowledge base article. Check with support to find out if they have a preference for how it is handled.